public/index.php

```diff
@@ -1,12 +1,9 @@
 <?php
 
-use Tempest\AppConfig;
-use Tempest\Application\Environment;
 use Tempest\Tempest;
-use function Tempest\env;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-Tempest::boot(__DIR__ . '/../')->http()->run();
+Tempest::boot(dirname(__DIR__))->http()->run();
```

Comment:
These are doing the same exact thing. What's the point in this refactor?

Reply:
The function doesn't mess with or assume directory separators, where appending the path segment does. It's a common recommendation i've seen, but i can revert it.

Reply:
I'm not opposed to the change necessarily, but let's move this to a separate PR so we can retain a single focus on this one.